### PR TITLE
Add DateTimeInput samples

### PR DESCRIPTION
**Pull request by MonikaKirkova**
Closes IgniteUI/igniteui-xplat-docs/issues/1057

### CODE REVIEW

File: samples/inputs/date-time-input/min-max-value/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    constructor(props: any) {
+        super(props);           
+        this.iconRef = this.iconRef.bind(this);     
+        this.dateTimeInputRef = this.dateTimeInputRef.bind(this);
+    }
+
+    public render(): JSX.Element {
+        return (
+            <div className="container sample">
+                <IgrDateTimeInput ref={this.dateTimeInputRef} max={new Date(2024, 6, 25)}>
```
Comment (RivaIvanova) [inline samples/inputs/date-time-input/min-max-value/src/index.tsx:22]: The sample uses the `min` and `max` properties while the code snippet included in the [docs](https://github.com/IgniteUI/igniteui-xplat-docs/pull/1154/files#diff-3405d20657edffbb2aa13af8509825cd27411739d8d0770f2922a8105468f83cR214-R224) shows the `minValue` and `maxValue`.

```tsx
public dateTimeInputRef(input: IgrDateTimeInput) {
    if (!input) { return; }
    input.minValue = new Date(2021, 0, 1);
}
```

```tsx
 <IgrDateTimeInput ref={this.dateTimeInputRef} maxValue={new Date(2024, 6, 25)}></IgrDateTimeInput>
```

File: samples/inputs/date-time-input/input-format-display-format/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "scripts": {
+    "start": "set PORT=4200 && react-scripts --max_old_space_size=10240 start",
+    "build": "react-scripts --max_old_space_size=10240 build ",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "dependencies": {
+    "igniteui-dockmanager": "1.14.2",
+    "igniteui-react": "18.6.0",
+    "igniteui-react-core": "18.6.0",
```
Comment (RivaIvanova) [inline samples/inputs/date-time-input/input-format-display-format/package.json:17]: Perhaps these packages should be updated to be aligned with the other samples in the repo:
```json
"igniteui-dockmanager": "1.14.3",
"igniteui-react": "18.6.1-alpha.0",
"igniteui-react-core": "18.6.1-alpha.0"
```
This applies for all the samples.

File: samples/inputs/date-time-input/input-format-display-format/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<!-- NOTE: do not change this file because it's auto re-generated from template: -->
+<!-- https://github.com/IgniteUI/igniteui-react-examples/tree/vnext/templates/sample/ReadMe.md -->
+
+This folder contains implementation of React application with example of Sizing feature using [Icon](https://www.infragistics.com/products/ignite-ui-react/react/components/general-getting-started.html) component.
```
Comment (RivaIvanova) [inline samples/inputs/date-time-input/input-format-display-format/README.md:4]: All the `README.md` files seem to be related to the `Icon` component. They should update automatically when [running all samples](https://github.com/IgniteUI/igniteui-react-examples?tab=readme-ov-file#running-all-samples).

File: samples/inputs/date-time-input/min-max-value/src/index.tsx (hunk 1 above)
Comment (MonikaKirkova) [reply samples/inputs/date-time-input/min-max-value/src/index.tsx:22]: The `maxValue` and `minValue` properties are deprecated and `max` and `min` should be used instead. I will update the code snippet to use `max` and `min`.

**Pull request merged**